### ISSUE TITLE
Add basic Data Structure programs in any language.

### ISSUE THREAD

**Issue by kamal-kaur04** [kamal-kaur04/Data-Structures#1]
If there is no folder for particular language, make a folder 

> src-_language_ (e.g. src-python, src-c etc). 

and then start adding programs. 
**Happy Contributing ❤️**

**Comment by hrishikeshathalye** [comment]
Would like to add code for ADT Stack using array in C to the repo. Would this be helpful?

**Comment by kamal-kaur04** [comment]
Yes you can. Please make a folder as per guidelines then add program.
Thanks.

**Comment by hrishikeshathalye** [comment]
I have added the code and created a PR. Do check it out.
